Tools/autotest/sim_vehicle.py

```diff
@@ -985,6 +985,9 @@ def generate_frame_help():
                  default=0,
                  type=int,
                  help="mavproxy module debug")
+group.add_option("", "--no-rcin",
```

Comment:
This really ought to be `--no-mavproxy-rc`.  Nothing stops something else connecting to the SITL RC-in port, right?

Reply:
true, but not really likely to be anything else